src/main/java/mekanism/common/item/interfaces/IItemHUDProvider.java

```diff
@@ -4,9 +4,16 @@
 import net.minecraft.network.chat.Component;
 import net.minecraft.world.entity.EquipmentSlot;
 import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ArmorItem;
 import net.minecraft.world.item.ItemStack;
 
 public interface IItemHUDProvider {
 
     void addHUDStrings(List<Component> list, Player player, ItemStack stack, EquipmentSlot slotType);
+
+    default void addCurioHUDStrings(List<Component> list, Player player, ItemStack stack) {
+        if (this instanceof ArmorItem armor) {
+            addHUDStrings(list, player, stack, armour.getSlot());
```

Comment:
Uhhh I feel like you didn't test if this compiles after changing the variable name.

Reply:
Oh my bad